packages/claims/package.json

```diff
@@ -29,27 +29,27 @@
   },
   "license": "GPL-3.0-or-later",
   "dependencies": {
-    "@ew-did-registry/did": "0.6.0",
-    "@ew-did-registry/did-document": "0.6.1",
-    "@ew-did-registry/did-ethr-resolver": "0.6.1",
-    "@ew-did-registry/did-ipfs-store": "0.6.0",
-    "@ew-did-registry/did-resolver-interface": "0.6.0",
-    "@ew-did-registry/did-store-interface": "0.6.0",
-    "@ew-did-registry/jwt": "0.6.0",
-    "@ew-did-registry/keys": "0.6.0",
+    "@ew-did-registry/did": "^0.6.0",
```

Comment:
Why is this? I guess we should let lerna to manage this. This also could lead to problems when latest will depend on canary